Apps/WebClient/src/ClientApp/src/store/modules/auth/mutations.ts

```diff
@@ -27,6 +28,7 @@ export const mutations: MutationTree<AuthState> = {
         Vue.set(state.authentication, "accessToken", undefined);
         Vue.set(state.authentication, "scopes", undefined);
         Vue.set(state.authentication, "idToken", undefined);
+        Vue.set(state.authentication, "identityProvider", "");
```

Comment:
may be use undefined instead of "", if "" is better to use here, we can get a task for refactoring better to be consistent. Also this should be aligned with the check in the isBcscLogin method in the getters.ts above.

Reply:
the type of the other variables is a string or undefined, idp is a string only.
To your second point, the authentication object could be undefined, but that is not the variable of said object
```
return state.authentication === undefined
```

Reply:
oh I see, my bad